platform/solr/solr-factory-impl/src/main/java/org/codice/solr/factory/impl/SolrCloudClientFactory.java

```diff
@@ -86,19 +86,22 @@ SolrClient createSolrCloudClient(String zookeeperHosts, String collection) {
       try {
         uploadCoreConfiguration(collection, client);
       } catch (SolrFactoryException e) {
-        LOGGER.debug("Solr({}): Unable to upload configuration to Solr Cloud", collection, e);
+        LOGGER.debug("Solr({}): Unable to upload configuration to SolrCloud", collection, e);
         return null;
       }
 
       try {
         createCollection(collection, client);
       } catch (SolrFactoryException e) {
-        LOGGER.debug("Solr({}): Unable to create collection on Solr Cloud", collection, e);
+        LOGGER.debug("Solr({}): Unable to create collection on SolrCloud", collection, e);
         return null;
       }
 
       client.setDefaultCollection(collection);
       return closer.returning(client);
+    } catch (LinkageError | Exception e) {
```

Comment:
Just double checking, a broad `Exception` was required here?

Reply:
I was not able to figure out what code was catching the escaping exceptions but wherever it was it is swallowing them based on the logs.  I know I needed the `LinkageError` to find a class def issue.  I am not sure about `Exception` but wanted to save the next person from wasting a lot of hours hunting down the swallowed exceptions like I did.